src/current/v25.2/architecture/transaction-layer.md

```diff
@@ -432,6 +432,45 @@ Despite their logical equivalence, the transaction coordinator now works as quic
 
 Additionally, when other transactions encounter a transaction in `STAGING` state, they check whether the staging transaction is still in progress by verifying that the transaction coordinator is still heartbeating that staging transaction's record. If the coordinator is still heartbeating the record, the other transactions will wait, on the theory that letting the coordinator update the transaction record with the result of the attempt to commit will be faster than going through the transaction status recovery process. This means that in practice, the transaction status recovery process is only used if the transaction coordinator dies due to an untimely crash.
 
+### Buffered writes
+
+{% include_cached new-in.html version="v25.2" %} Buffered Writes enhance transaction throughput and reduce operational cost by minimizing the number of round-trips between the client and the database during write operations.
```

Comment:
`client` here might be a bit of a misnomer. You might say "between the gateway node and other nodes during..." or "between the transaction coordinator and other nodes during..."

I note now we use client-side below as well. Perhaps that makes sense in the context of this document, I haven't read through the larger document in a while.

Reply:
makes sense - i updated to say between gateway node and others

i think the other use(s) of 'client' in this new section can also be updated, generally we use the term gateway node in the architecture docs